chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/Permissions.java

```diff
@@ -125,4 +125,12 @@ public boolean hasAccess(final String domain, final String action, final Object
         }
     }
 
+
+    public boolean hasAccess(final String domain, final String action, final Object obj, final Object obj2) {
+        if (domainPermissions.containsKey(domain)) {
+            return domainPermissions.get(domain).hasAccess(action, obj, obj2);
+        } else {
```

Comment:
This may be pedantic, and therefore is not something you _need_ to change, but, the else statement is not required in this case

Reply:
While that's true, it's also consistent with the way the other `hasAccess` methods are structured, so if I _did_ do it that way I'd feel required to change all of the other ones too

Reply:
Yeah I see that now too - approved the PR (not that I would have held it back over this anyway)